### DISCUSSION TITLE
Feature to only delete certain user comments

### DISCUSSION THREAD

**Discussion by jomarcenter-mjm** [ThioJoe/YT-Spammer-Purge#44] [Ideas]
I would like to request to add some additional information from only deleting comment from a certain user account.

**Comment by KendallDoesCoding** [comment]
> 

You can already do that right? 

Wdym by this?

**Reply by TechStudent10** [reply]
This was a month ago. It has probably been implemented in the past month.

**Comment by KendallDoesCoding** [comment]
Ok cool.
